app/controllers/v1/tasks_controller.rb

```diff
@@ -56,6 +56,18 @@ def complete
     end
   end
 
+  def cancel_correction
```

Comment:
We have written a request spec for the successful case (uploading soon). We can't think of how this would ever not be successful, so unsure how to test for it. Does this mean we don't want to catch for errors in that case? I think we currently have several actions that have untested and hopefully impossible unhappy paths.

Reply:
Scrap that, your last message gives us a good clue on how to do this. (james speaking as Cristina on both of these btw)